src/strtab.rs

```diff
@@ -91,6 +91,23 @@ impl<'a> Strtab<'a> {
     #[cfg(feature = "alloc")]
     /// Converts the string table to a vector of parsed strings.
     ///
+    /// Note: If you want to get the correct `Vec` of shdr_strtab, don't use this method but use `get_at` to get it sequentially.
```

Comment:
Can you remove the comment about the section header strtab, as philipc said this applies to any strtab and primary use of this method is just to see the contents it parsed